BUILDING.md

```diff
@@ -240,7 +240,7 @@ test with Python 3.
 * GNU Make 3.81 or newer
 * Python (see note above)
   * Python 2.7
```

Comment:
Legacy Python?!?

Reply:
This is Node.js 14, which goes End-of-Life in April 2023. We dropped legacy Python for Node.js 16 but 14 was the last release where we supported building with both legacy Python and Python 3 -- this PR merely allows the scripts to use Python 3.11.